kong/templates/nginx_kong.lua

```diff
@@ -439,13 +439,11 @@ server {
         }
     }
 
-> if not legacy_hybrid_protocol then
```

Comment:
Seems we are missing the gate here and not handling the switch at `_M:handle_cp_websocket`?

Reply:
Hmm? Sorry, I don't quite follow.

Reply:
oh never mind, I am dumb :man_facepalming: